### ISSUE TITLE
Integrate setPreprocessor with Engine

### ISSUE THREAD

**Issue by farmaazon** [enso-org/ide#1020]
### Summary
After #1019 the GraphEditor should have frp enpoint informing about new expression for given visualization. The event should set a new expression using the `executionContext/modifyVisualization` method of Language Server.

### Acceptance Criteria & Test Cases
* A debug shortcut in Histogram which set alternative expression discarding the input data and returning some hard-coded array serialized to json.

**Comment by kustosz** [comment]
The current documentation (https://github.com/enso-org/ide/blob/main/src/rust/ide/view/graph-editor/src/component/visualization/foreign/java_script/definition.rs#L55) describes `setPreprocessor` as taking a code string.
However, I find it important to also handle the second parameter exposed by the backend – the evaluation module:
https://dev.enso.org/docs/enso/language-server/protocol-language-server.html#visualisationconfiguration

This is important to ensure visualizations can be created independent of the data definitions.
If you don't want to always pass this argument, a sensible default would be the module `Main` of the project defining the visualizations (not the current project! the project that defines the visualisation). However, an option to override should be available.

<blockquote><img src="https://avatars.githubusercontent.com/u/17415943?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/enso-org/ide">enso-org/ide</a></strong></div><div>Enso – a visual and textual functional programming language. - enso-org/ide</div></blockquote>
<blockquote><img src="/gitbook/images/favicon.ico" width="48" align="right"><div><strong><a href="https://dev.enso.org/docs/enso/language-server/protocol-language-server.html#visualisationconfiguration">Enso Protocol Language Server Message Specification · Enso Developer Documentation</a></strong></div><div>The developer documentation for the Enso compiler, runtime, and IDE..
</div></blockquote>

**Comment by farmaazon** [comment]
This issue I will deliver as it is described, without adding a module parameter - for that I've created a new issue https://github.com/enso-org/ide/issues/1167

**Comment by farmaazon** [comment]
The debug shortcut Ctrl+D is available in Histogram. It sets the preprocessor to `x -> [1,2,3,4]`
